notifications_utils/logging.py

```diff
@@ -16,46 +16,60 @@
 logger = logging.getLogger(__name__)
 
 
+def build_log_line(extra_fields):
+    fields = []
+    if 'service_id' in extra_fields:
+        fields.append(str(extra_fields.get('service_id')))
+    fields += [extra_fields.get('method'), extra_fields.get('url'), str(extra_fields.get('status'))]
```

Comment:
i'd prefer these to be `[` `]` - otherwise it'll crash when trying to `join` and a KeyError here would be much easier to debug than `TypeError: sequence item 1: expected str instance, NoneType found`